disabled plugin install from vdr source, we dont support this way
patch to support LINGUAS variable
fixed install on multicore cpu, creating CONFDIR on later time
install libsi includes in own dir
Signed-of-by: Joerg Bornkessel <hd_brummy@gentoo.org> 20/Jan/2015
diff -Naur vdr-2.0.6.orig/Makefile vdr-2.0.6/Makefile
--- vdr-2.0.6.orig/Makefile	2014-09-22 22:17:23.000000000 +0200
+++ vdr-2.0.6/Makefile	2014-09-24 15:22:43.000000000 +0200
@@ -109,7 +109,7 @@
 VDRVERSION = $(shell sed -ne '/define VDRVERSION/s/^.*"\(.*\)".*$$/\1/p' config.h)
 APIVERSION = $(shell sed -ne '/define APIVERSION/s/^.*"\(.*\)".*$$/\1/p' config.h)
 
-all: vdr i18n plugins
+all: vdr i18n
 
 # Implicit rules:
 
@@ -162,7 +162,7 @@
 
 PODIR     = po
 LOCALEDIR = locale
-I18Npo    = $(wildcard $(PODIR)/*.po)
+I18Npo    = $(foreach dir,$(LINGUAS),$(wildcard $(PODIR)/$(dir)*.po))
 I18Nmo    = $(addsuffix .mo, $(foreach file, $(I18Npo), $(basename $(file))))
 I18Nmsgs  = $(addprefix $(LOCALEDIR)/, $(addsuffix /LC_MESSAGES/vdr.mo, $(notdir $(foreach file, $(I18Npo), $(basename $(file))))))
 I18Npot   = $(PODIR)/vdr.pot
@@ -185,7 +185,7 @@
 
 install-i18n: i18n
 	@mkdir -p $(DESTDIR)$(LOCDIR)
-	cp -r $(LOCALEDIR)/* $(DESTDIR)$(LOCDIR)
+	@(cd $(LOCALEDIR); for linguas in $(LINGUAS); do [ "$$linguas" = "en" ] && continue; cp -r --parents $$linguas* $(DESTDIR)$(LOCDIR); done)
 
 # The 'include' directory (for plugins):
 
@@ -247,7 +247,7 @@
 
 # Install the files (note that 'install-pc' must be first!):
 
-install: install-pc install-bin install-dirs install-conf install-doc install-plugins install-i18n install-includes
+install: install-pc install-bin install-dirs install-conf install-doc install-i18n install-includes
 
 # VDR binary:
 
@@ -259,11 +259,12 @@
 
 install-dirs:
 	@mkdir -p $(DESTDIR)$(VIDEODIR)
-	@mkdir -p $(DESTDIR)$(CONFDIR)
-	@mkdir -p $(DESTDIR)$(CACHEDIR)
+#	@mkdir -p $(DESTDIR)$(CONFDIR)
+#	@mkdir -p $(DESTDIR)$(CACHEDIR)
 	@mkdir -p $(DESTDIR)$(RESDIR)
 
 install-conf:
+	@mkdir -p $(DESTDIR)$(CONFDIR)
 	@cp -pn *.conf $(DESTDIR)$(CONFDIR)
 
 # Documentation:
@@ -290,8 +291,11 @@
 # Includes:
 
 install-includes: include-dir
-	@mkdir -p $(DESTDIR)$(INCDIR)
-	@cp -pLR include/vdr include/libsi $(DESTDIR)$(INCDIR)
+#	@mkdir -p $(DESTDIR)$(INCDIR)
+#	@cp -pLR include/vdr include/libsi $(DESTDIR)$(INCDIR)
+	@mkdir -p $(DESTDIR)$(INCDIR)/vdr $(DESTDIR)$(INCDIR)/vdr/libsi
+	@cp -pLR include/vdr $(DESTDIR)$(INCDIR)
+	@cp -pLR include/libsi Make.config $(DESTDIR)$(INCDIR)/vdr
 
 # pkg-config file:
 
